src/tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -320,7 +320,6 @@ private bool ExecuteInternal ()
 
         config.Assets.Add(new VfsEntry ("dotnet.timezones.blat") { VirtualPath = "/usr/share/zoneinfo/"});
         config.Assets.Add(new WasmEntry ("dotnet.wasm") );
-        config.Assets.Add(new CryptoWorkerEntry ("dotnet-crypto-worker.js") );
```

Comment:
should we also remove the definition of the `CryptoWorkerEntry` class?

Reply:
Yep! Missed that one.